test/build_service.bats

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bats
+
+load 'test_helper/debug'
+load 'test_helper/bats-support/load'
+load 'test_helper/bats-assert/load'
+load 'test_helper/workspace'
+
+BUILD_SERVICE_DIR="${PWD}/test/test_yamls/build"
+
+function setup() {
+	add_dir_with_ws ${BUILD_SERVICE_DIR} tarantino/build_service.yml
```

Comment:
Ah didn't notice you actually needed that subfolder on the yaml file... yeah remove that last commit I suggested and go for the merge

Reply:
This new GitHub feature makes hitting apply from mobile too easy. Going to need to be more responsible in the future. I think the last four commits were probably the worst commit messages I've ever written ...

Reply:
Now, if only I could do a mobile revert. Hahaha!

Reply:
haha they actually go down as my commits, yeah it's a cool feature but probably needs a bit more meat to be fully useful (you also can't suggest anything to replace multiple lines - single only)

lol I did the suggest from mobile too.. we are both locked out until we get to desktop